packaging/hudi-spark-bundle/pom.xml

```diff
@@ -248,6 +260,13 @@
         <spark.bundle.hive.shade.prefix>org.apache.hudi.</spark.bundle.hive.shade.prefix>
       </properties>
     </profile>
+    <profile>
+      <id>spark-shade-unbundle-avro</id>
+      <properties>
+        <spark.bundle.avro.scope>provided</spark.bundle.avro.scope>
+        <spark.bundle.spark.shade.prefix>/</spark.bundle.spark.shade.prefix>
```

Comment:
@lamber-ken I see a '/' here for the property "spark.bundle.spark.shade.prefix" if the profile is activated. Is this intended? Does this mean shade with empty prefix (same as not shading)? I dont know much about profiling either so wanted to clarify with you.

Reply:
Good catch